pyproject.toml

```diff
@@ -27,3 +27,14 @@ exclude = '''
   )/
 )
 '''
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+line_length = 88
+known_tests = "tests"
+skip_glob = "**/*_pb2.py"
+order_by_type = true
+sections = "FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,TESTS,LOCALFOLDER"
```

Comment:
Luckily, `black` and `isort` don't seem to interfere with each other (yet). So the only really important config value here is `line_length = 88`. That's what we configured for `black`, so `isort` gets a matching value. And maybe `skip_glob`, so that we don't try to format auto-generated files.

The rest of those is just personal preference. `sections` especially just tells `isort` in which order imports of different origin are to be sorted.

Reply:
Does the `section` configuration differ from the default?

Reply:
Nope, default is `"FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER"` (see [here](https://pycqa.github.io/isort/docs/configuration/options/#sections)). The only difference is the `TEST` section, we could revert to the default if wanted.

Reply:
So if `TEST` is usually a part of the `STDLIB` I would actually stick to the default here.

Reply:
Actually `TEST` seems to be a custom section (hooray for copying config files around), see e91fcc8. The default is good enough for us.